index.html

```diff
@@ -126,6 +132,96 @@ <h2 class="connect">LET’S CONNECT</h2>
 
 </section>
 
+<section class="works-desktop desktop">
+  <article class="work1">
+    <img src="images/work1-desktop-img.svg" alt="Work image">
+    <div class="work-text-container">
+      <header class="work-header">Tonic</header>
+      <ul class="work-subheader">
+        <li class="list1">CANOPY</li>
+        <li class="dot"></li>
+        <li class="list2">Back End Dev</li>
+        <li class="dot"></li>
+        <li class="list2">2015</li>
+      </ul>
+      <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] You should not duplicate your HTML code for the sole purpose of rendering it in different screen sizes. That is the job of CSS not HTML. Thus, kindly remove the duplicated code for desktop and have only one WORKS section HTML code and style it using CSS media query so that the layout will adapt to different screen sizes.
![image](https://user-images.githubusercontent.com/44057863/178073566-64609a0d-ea64-4b6b-a8ba-d59393a9e6cf.png)

- [ Optional ] The background color for the `works` section is same as the `body` which is `#e5e5e5`.

Reply:
@gtekle It would be nice if all of these changes were brought to me on my first code review instead of having to spend all my reviews doing the changes suggested only to get different things to implement every time. I would be more than glad to push them all in a single review! Just a thought.

Reply:
I completely agree with you 💯 and I apologize for the inconvenience. 
You are making good progress in addressing all the requested changes, though. Keep pressing forward and happy coding 👍

Reply:
No need to apologize at all! I find your feedback very helpful, that also motivated me to clean all my CSS code while I was at it. It took a while but my CSS looks decent now! Good practice always pays off!